src/OpenLoco/GameCommands/VehiclePickup.cpp

```diff
@@ -0,0 +1,68 @@
+#include "../Interop/Interop.hpp"
+#include "../Management/Expenditures.h"
+#include "../Types.hpp"
+#include "../Vehicles/Vehicle.h"
+#include "GameCommands.h"
+
+using namespace OpenLoco::Interop;
+
+namespace OpenLoco::GameCommands
+{
+    static loco_global<uint8_t, 0x009C68EA> gGameCommandExpenditureType; // premultiplied by 4
```

Comment:
perhaps we should make an accessor function for this :P.

Yes, finally I can throw that comment out instead of receive it!

Reply:
We absolutely should! 😄